src/api/java/com/minecolonies/api/tileentities/TileEntityColonyBuilding.java

```diff
@@ -514,4 +522,52 @@ public AbstractContainerMenu createMenu(final int id, @NotNull final Inventory i
     {
         return new ContainerBuildingInventory(id, inv, colonyId, getBlockPos());
     }
+
+    /**
+     * Reactivate the hut of this tileEntity.
+     * Load the schematic data and set the style correctly.
+     */
+    public void reactivate()
+    {
+        final List<String> tags = new ArrayList<>(this.getPositionedTags().get(BlockPos.ZERO));
+        tags.remove("deactivated");
+        if (!tags.isEmpty())
+        {
+            this.setStyle(tags.get(0));
```

Comment:
order of tags is not guaranteed, so may want to actually check the right one out instead

Reply:
I rely on the hut blocks having only 2 tags here. the deactivated and the other, there is no other way to detect it else

Reply:
could've given it a style prefix